Common.mk

```diff
@@ -570,7 +573,9 @@ endif
 #### Image Helpers
 
 ifneq ($(IMAGE_NAMES),)
-.PHONY: local-images images
+.PHONY: local-images-w-helm images-w-helm local-images images
```

Comment:
I would probably not include these extra targets, instead just have people run helm/build or helm/push depending on what they are doing. If you think there is enough of a use case here that others will use these helper targets, its totally fine to leave them. We could always remove them later if we dont hear about anyone using them

Reply:
I talked to our package team today (as I know a few frequent users that build images and helm charts together) and they seemed to be ok with the idea to keep them as separated targets. So I have removed the additional local-images-w-helm images-w-helm targets!